sdk/python/feast/infra/gcp.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, config: RepoConfig):
         self._namespace = config.online_store.namespace
 
         assert config.offline_store is not None
+        self._write_concurrency = config.online_store.write_concurrency
+        self._write_batch_size = config.online_store.write_batch_size
```

Comment:
nit: move these up by 2 lines so that online store & offline store code are logically grouped

Reply:
rebase gone wrong

Reply:
done